documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_ValueOnly.adoc

```diff
@@ -186,7 +214,20 @@ For a _SubmodelElementList,_ the struct is serialized as array denoted by square
 }
 ----
 
-For a _MultiLanguageProperty_ named “Label”, the payload is minimized to the following:
+If the _SubmodelElementList_ itself is the subject for ValueOnly the serialization is minimized to the following:
+
+[source,json,linenums]
+----
+ [
```

Comment:
FYI, previously it stated that everything is serialized as a JSON object - this is not a JSON object but a JSON array. They are considered different things in JSON.